test/integration/test_project_migrate.c

```diff
@@ -895,7 +909,7 @@ int all_tests() {
 	mu_run_test(test_load_v9_v10_stack_vars_sp, 9, "prj/v9-sp-vars.rzdb");
 	mu_run_test(test_load_v9_v10_stack_vars_bp, 10, "prj/v10-bp-vars.rzdb");
 	mu_run_test(test_load_v9_v10_stack_vars_sp, 10, "prj/v10-sp-vars.rzdb");
-	mu_run_test(test_load_v12);
```

Comment:
This change is wrong? Please don't remove the old test

Reply:
OK, just a mistake.